pkg/util/utils.go

```diff
@@ -652,3 +660,120 @@ func ValidateSysctls(strSlice []string) (map[string]string, error) {
 	}
 	return sysctl, nil
 }
+
+func DefaultContainerConfig() *config.Config {
+	return defaultContainerConfig
+}
+
+func containerConfig() *config.Config {
```

Comment:
I think this should be an `init()` to make it explicit that this is happening whenever the package is used.

Reply:
I haven't heard of that yet. Do you have link? It's mentioned in [effective go](https://golang.org/doc/effective_go.html#initialization).

I prefer having such code in an `init()` mostly because of the `os.Exit(...)`. I find it more idiomatic for such cases. I wouldn't care as much without the exit.

Reply:
I thought one of the linters was preventing it, I will try again.